targets/TARGET_Cypress/TARGET_PSOC6/cy_qspi_api.c

```diff
@@ -38,7 +38,8 @@ qspi_status_t qspi_free(qspi_t *obj)
 
 qspi_status_t qspi_frequency(qspi_t *obj, int hz)
 {
-    return CY_RSLT_SUCCESS == cyhal_qspi_set_frequency(&(obj->hal_qspi), (uint32_t)hz) ? QSPI_STATUS_OK : QSPI_STATUS_ERROR;
+    /* Return OK since this API is not implemented in cy_hal */
+    return QSPI_STATUS_OK;
```

Comment:
` QSPI_STATUS_ERROR` here

Reply:
TESTS/mbed_hal/qspi/main.cpp line 261 will fail:
ret = qspi_frequency(&qspi.handle, frequency);

What should we do in this case?

Reply:
I see, this function should always return TRUE as it should be implemented 🙄 

Lets leave it as it is, this should get fixed once we have capabilities